.eslintrc.json

```diff
@@ -77,5 +77,19 @@
         "prefer-const": ["error"],
         "no-var": "error",
         "eqeqeq": "error"
-    }
+    },
+    "overrides": [
+        {
+            "files": [
+                "tests/**/*.js"
+            ],
+            "env": {
+                "jest": true,
+                "node": true
+            },
+            "rules": {
+                "max-len": ["warn", { "code": 100, "tabWidth": 2 }]
```

Comment:
I'm still not a big fan of this. (I wanted to say it once again°

Reply:
Ok, I'll revert to error :)